components/urlbox_io/common/utils.mjs

```diff
@@ -0,0 +1,38 @@
+import hmacSha1 from "crypto-js/hmac-sha1.js";
+import qs from "qs";
+
+export const toQueryString = (options) => {
+  return qs.stringify(options, {
```

Comment:
Could you please comment on this function saying why you are doing this? Just to help some other people that may give maintenance to this code in the future